semih.js

```diff
@@ -0,0 +1,119 @@
+/*Assignment #1*/
+/*printing hello world in a couple of languages with console.log function */
+console.log("Hello World // English");
+console.log("Merhaba Dünya // Turkish");
+console.log("Привет, мир // Russian");
+console.log("Hallo Wereld // Dutch");
+
+
+/*Assignment #2*/
+/*added \ for escape character*/ 
+console.log('I\'m awesome'); 
+
+
+/*Assignment #3*/
+let x;
+console.log("The value of my variable x will be: 'undefined'");
+console.log(x);
+x = 0;
+console.log("The value of my variable x will be: 0");
+console.log(x);
+
+/*Assignment #4*/
+let y = "some_string";
+console.log("The value of my variable y will be: 'some_string'");
+console.log(y);
+y = "new_string"
+console.log("The value of my variable y will be: 'new_string'");
+console.log(y);
+
+
+/*Assignment #5*/
+let z = 7.25;
+console.log(z);
+let a = Math.round(z);
+console.log(a);
+let highestvalue;
+
+if (a<z) {
+    highestvalue=z;
+} else {
+    highestvalue=a;
+}
+
+console.log(highestvalue);
```

Comment:
This is correct. However, I would like you to explore three things here:
1. You can use `const` keyword for both `z` and `a` variables instead of `let` since we aren't reassigning them to any new values.
2. Can you learn about the difference between `Math.floor` and `Math.round` methods 
3. You can also use `Math.max` method to select a maximum value from `a` and `z` instead of the `if-else` block

Reply:
I'll make sure I get used to using `const` over `let`.
Regarding the `Math.floor` & `Math.round`, I checked the MDN and as it is also stated there `Math.floor` returns the largest integer less than the given one. On the other hand, `Math.round` returns the nearest integer. So it could be higher or lower than the given one. 

example: `console.log(Math.floor(0.999))=0` and `console.log(Math.round(0.999))=1`

`Math.max` is also very useful and makes `if-else` unnecessary in our case as you mentioned. Thanks a lot for the additional info!